Makefile

```diff
@@ -44,6 +44,9 @@ test:
 	# run the tests
 	@go test  $(PKGS)
 
+test-e2e:
+	go test -timeout 55m -v ./test/e2e/ $(TEST_RUN_ARGS) --kubeconfig=$(KUBECONFIG)
```

Comment:
Like discussed today, it is probably a good idea to disable caching here.

Reply:
Already added that in another branch :relaxed: